aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -41,7 +41,7 @@ impl<T: std::fmt::Debug> PanicOr<T> {
 
 pub fn code_invariant_error<M: std::fmt::Debug>(message: M) -> PanicError {
     let msg = format!(
-        "Delayed materialization code invariant broken (there is a bug in the code), {:?}",
+        "Code invariant broken (there is a bug in the code), {:?}",
```

Comment:
@igor-aptos do you think this is reasonable (and should I then move it elsewhere) - we've already used it for general (non-delayed-field) errors, or maybe delayed fields meant things that were developed alongside delayed fields. I could also create another macro somewhere, but then this should also be delayed_field_invariant_error or smt.

Reply:
I think making this general is perfectly fine

Reply:
we also have it defined in two places (aptos-aggregator and third-party move-vm), that don't link to each other :/ 

maybe some parent place for both

Reply:
moved as much as I could to types/errors, unfortunately we have a specific type included in crates/ and moving even more led to circular dependencies, but should be a decent start?